sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-classification-multiclass-task-sentiment-analysis/automl-nlp-multiclass-sentiment-mlflow.ipynb

```diff
@@ -318,6 +326,94 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.3 Runs with models from Hugging Face (Preview)\n",
+    "\n",
+    "In addition to the model algorithms supported natively by AutoML, you can launch individual runs to explore any model algorithm from HuggingFace transformers library that supports image classification. Please refer to this [documentation](https://huggingface.co/models?pipeline_tag=text-classification&library=transformers&sort=trending) for the list of models.\n",
+    "\n",
+    "While you can use any model from Hugging face to support this task, we have curated a set of model algorithms in our registry. We provide a set of sensible default hyperparameters for these model algorithms. You can fetch the list of curated model algorithms using code snippet below."
```

Comment:
this doesn't really apply, as the models won't come from the registry and also this will list models that the user cannot use - like for instance llama

Reply:
Removed model listing from azureml-staging registry